src/backend/services/StateManager.ts

```
@@ -0,0 +1,196 @@
+      throw error;
+    }
+  }
+
+  /**
+   * Create a new scenario
+   */
+  public async createScenario(scenarioData: any): Promise<string> {
+    const session = this.client.startSession();
+    try {
+      let scenarioId: string = '';
+      
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)